scripts/db-race-conditions-fix.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+docker compose up -d database
```

Comment:
Would be nice if we can figure out what's going on to avoid needing this script, but this is OK for now.
Should we document it in the README in a troubleshooting section?

Reply:
yeah I spoke with Thomas from pre-award and he said they use a script similar to this on windows because of race conditions with initialising the db... made it sound like it's just a windows problem.

Good shout I'll add to the README